docs/src/test/scala/docs/http/scaladsl/server/directives/FutureDirectivesExamplesSpec.scala

```diff
@@ -54,6 +54,10 @@ class FutureDirectivesExamplesSpec extends RoutingSpec with CompileOnlySpec {
   }
 
   "onCompleteWithBreaker" in {
+    /*
```

Comment:
we could maybe use the `ignore` scalatest thing rather then commenting-out... but this is fine.

Reply:
Yes, of course, thanks!